packages/forms-demo/src/components/Fields/Input/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {connect} from 'cerebral/react'
+import {css} from 'aphrodite'
```

Comment:
Haven't explored this lib yet. Are this really makes much sense for demo?
Isn't just css not enough?